root.tf

```diff
@@ -691,6 +717,46 @@ module "s3_vpc_endpoint" {
   )
 }
 
+module "github_oidc_provider" {
+  source      = "./tdr-terraform-modules/identity_provider"
+  audience    = "sts.amazonaws.com"
+  thumbprint  = "6938fd4d98bab03faadb97b34396831e3780aea1"
+  url         = "https://token.actions.githubusercontent.com"
+  common_tags = local.common_tags
+}
+
+module "consignment_api_github_environment" {
+  source          = "./tdr-terraform-modules/github_environments"
+  environment     = local.environment
+  repository_name = "nationalarchives/tdr-consignment-api"
+  team_slug       = "transfer-digital-records-admins"
+  secrets = {
+    TITLE_STAGE            = title(local.environment)
+    ACCOUNT_NUMBER         = data.aws_caller_identity.current.account_id
+    MANAGEMENT_ACCOUNT     = data.aws_ssm_parameter.mgmt_account_number.value
+    SLACK_FAILURE_WORKFLOW = data.aws_ssm_parameter.slack_failure_workflow.value
+    SLACK_SUCCESS_WORKFLOW = data.aws_ssm_parameter.slack_success_workflow.value
+    WORKFLOW_PAT           = data.aws_ssm_parameter.workflow_pat.value
+  }
+}
+
+module "e2e_tests_github_environment" {
```

Comment:
Maybe rename slightly to make clearer what this module is: `github_e2e_tests_environment`